### ISSUE TITLE
Do you upload Pre-Trained Model?

### ISSUE THREAD

**Issue by humayun** [foamliu/InsightFace-v2#17]
Dear,

Question 1: Are you planning or already upload the pre-trained models (if you already trained multiple model)? Can you share the links on github to download those model and used for only inference and validation?

Question 2: If you can share the hyperparameters for training each specific model with results and loss function? Also, if you can share how much time it will take to train those models on your specific machine?

Best,

**Comment by foamliu** [comment]
The pre-trained models can be found here: https://github.com/foamliu/InsightFace-v2/releases/download/v1.0/BEST_checkpoint_r101.tar.

**Comment by iwanna-sonhs** [comment]
Hello, 
I run with your checkpoint https://github.com/foamliu/InsightFace-v2/releases/download/v1.0/BEST_checkpoint_r101.tar.
Accuracy: 99.61666666666666%, threshold: 75.49033778039826

Different with your report, why ?
